platzky_redirections/plugin.py

```
@@ -93,14 +67,32 @@ def setup_routes(app, redirections):
 
 
 def redirect_with_name(destination, code, name):
+    """Create a named redirect function for use as a Flask view."""
+
     def named_redirect(*args, **kwargs):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `named_redirect`

(ANN202)